### DISCUSSION TITLE
Would it be possible to update the conan recipe and version?

### DISCUSSION THREAD

**Discussion by gprosch** [ImageMagick/ImageMagick#6858] [Development]
### ImageMagick version

7.0.11-14

### Operating system, version and so on

Windows

### Description

The ImageMagick library version on Conan is 7.0.11-14 dated 5-30-2022 and the recipe is for Conan 1.x.  Would it be possible to update to the latest version of ImageMagick and also to update the recipe to be Conan 2.x compatible?

**Comment by urban-warrior** [comment]
We oversee the management of ImageMagick's source code and Windows binary distributions. However, it's important to note that all other distributions are handled by external teams, and we do not have influence or control over their management processes.
